src/services/app/user.ts

```diff
@@ -26,6 +28,8 @@ export default class UserService {
 
     private static profileContentStorage = new ProfileContentStorage();
 
+    public static hubspotClient = new Client({ apiKey: config.hubspotKey });
```

Comment:
maybe? :thinking: 
```suggestion
    private static hubspotClient = new Client({ apiKey: config.hubspotKey });
```

Reply:
in this case, I used public because I need to import this "hubspotClient" on user.test